test/language/statements/class/elements/private-getter-brand-check-multiple-evaluations-of-class-realm.js

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2019 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Every new evaluation of a class creates a different brand (private getter)
+esid: sec-privatefieldget
+info: |
+  ClassTail : ClassHeritage { ClassBody }
+    ...
+    11. Let proto be ObjectCreate(protoParent).
+    ...
+    31. If PrivateBoundIdentifiers of ClassBody contains a Private Name P such that the P's [[Kind]] field is either "method" or "accessor",
+      a. Set F.[[PrivateBrand]] to proto.
+    ...
+
+  PrivateBrandCheck(O, P)
+    1. If O.[[PrivateBrands]] does not contain an entry e such that SameValue(e, P.[[Brand]]) is true,
+      a. Throw a TypeError exception.
+features: [class, class-methods-private]
+flags: [noStrict]
+---*/
+
+let eval1 = $262.createRealm().global.eval;
+let eval2 = $262.createRealm().global.eval;
+
+let classStringExpression = `(
+class {
+    get #m() { return 'test262'; }
+
+    access(o) {
+      return o.#m;
+    }
+  }
+)`;
+
+let createAndInstantiateClass = function (eval) {
```

Comment:
nit pick: to clarify it's a custom indirect eval, you might rename this to something else, e.g. `_eval`.